UI/TechTreeWnd.cpp

```diff
@@ -472,6 +475,16 @@ void TechTreeWnd::TechTreeControls::LDrag(const GG::Pt& pt, const GG::Pt& move,
     }
 }
 
+void TechTreeWnd::TechTreeControls::SetStatusState(TechStatus status, bool state) {
+    auto control_it = m_status_buttons.find(status);
+    if (control_it == m_status_buttons.end()){
```

Comment:
you don't use `control_it` so could do just `if (m_status_buttons.find(status) == m_status_buttons.end())`